test/cmdlineTests/yul_optimizer_steps_with_cleanup_sequence/input.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+pragma abicoder v2;
+
+contract C
+{
+    constructor() {}
+}
```

Comment:
Why not use the same code as in the other example so that we know exactly what to expect with that sequence?

Also, might be a good idea to use a cleanup sequence that's not the default so that the effects are different.

Reply:
This is tested in `yul_optimizer_steps_with_empty_optimization_sequence`, where we have an empty optimisation sequence (so just the hardcoded steps) with a specified and non-default clean-up sequence (`iDu`). This particular test was originally meant to be more of a sequence parsing tests than anything else, but now that I have those as unit tests (`CommandLineParser`), perhaps we can even delete this one?

Or can you suggest a specific optimisation sequence with a specific input that would make sense for this?

Reply:
Yeah, I guess you could remove some of the CLI tests that overlap with the unit tests you added. A lot of them were added to test parsing as well.